src/Compilers/Core/MSBuildTask/Microsoft.CSharp.Core.targets

```diff
@@ -36,7 +36,8 @@
                   $(ResolvedCodeAnalysisRuleSet);
                   @(AdditionalFiles);
                   @(EmbeddedFiles);
-                  @(EditorConfigFiles)"
+                  @(EditorConfigFiles);
+                  $(GeneratedFilesOutputPath)"
```

Comment:
Is this intended? This is a input list of _files_, so I assume we wouldn't specify a directory name because that might always cause the up to date check to say 'false'. Unless a directory existence is also handled?